integrations/langfuse/src/haystack_integrations/components/connectors/langfuse/langfuse_connector.py

```diff
@@ -112,6 +114,9 @@ def __init__(
             only accessible to the Langfuse account owner. The default is `False`.
         :param public_key: The Langfuse public key. Defaults to reading from LANGFUSE_PUBLIC_KEY environment variable.
         :param secret_key: The Langfuse secret key. Defaults to reading from LANGFUSE_SECRET_KEY environment variable.
+        :param httpx_client: Optional custom httpx.Client instance to use for Langfuse API calls. Note that this client
+            is not serializable, so when recreating a pipeline from YAML, any custom client passed here will not be
+            preserved - a new default client will be used instead.
```

Comment:
I'm not sure if this is what actually happens: my impression is that since the client is not serialized, the recreated Pipeline won't have this parameter. Could you verify?

Reply:
But that's what the doc says, no?

Reply:
Maybe put NOT in capital...

Reply:
ah you are referring to the Langfuse default client.
What I want to make clear is that in serde the HTTPX client is destroyed and never recreated...

Reply:
Got it, how about https://github.com/deepset-ai/haystack-core-integrations/pull/1308/commits/357e5b2d05fec863f166a8299cd042fbcbd005b6 ?